ifrs17/Import/ImportScopeCalculation.ipynb

```diff
@@ -660,6 +660,13 @@
                 "\n    private double nonPerformanceRiskRate => GetStorage().GetNonPerformanceRiskRate(Identity.Id);",
                 "\n        ",
                 "\n    double[] DiscountedCashflow.Values => Multiply(-1d, NominalValues.ComputeDiscountAndCumulateWithCreditDefaultRisk(MonthlyDiscounting, nonPerformanceRiskRate)); // we need to flip the sign to create a reserve view",
+                "\n}",
+                "\n",
+                "\npublic interface NEW_SCOPE : DiscountedCashflow",
```

Comment:
moving this to NominalCashflow the name could become: 
ProjectionBopNominalCashflow

Reply:
@dcolleoni Done.